tests/behat/behat_sitsgradepush.php

```diff
@@ -280,6 +285,22 @@ public function the_course_is_regraded(string $shortname): void {
         global $DB;
         $course = $DB->get_record('course', ['shortname' => $shortname]);
         grade_regrade_final_grades($course->id);
+
+        // Wait for the grade item to be updated. Check every 10 seconds, timeout in 300 seconds.
```

Comment:
Not sure if I looped you in on that, https://tracker.moodle.org/browse/MDL-70854 has now been integrated, 
https://tracker.moodle.org/browse/MDL-81714 (which we have an early draft of in our build) will make use of that to pull through the recalculation state and improve the wording where the formulae need fixing.

Though both of these will be 4.5+ and we will stay on 4.4 and then upgrade to 5.0(4.6).

We probably will want to revisit that once they have been integrated into core.

Reply:
see comment on https://github.com/ucl-isd/moodle-core/commit/e50522508ef26a3a09fb4bcefa85ee09da77d8a9#r144757264